.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
No, Idon't think so, because this would make it a lot more complicated. I moved the eslint and also prettier stuff from both client and server to the root folder, so this is already a simplified version I think. There also wasn't a change in logic, only the files were moved to the root folder.

Reply:
Sounds fine to me to do that in one PR, otherwise, it'll become ugly with git.